src/oaklib/utilities/mapping/cross_ontology_diffs.py

```diff
@@ -18,6 +17,12 @@
 from oaklib.utilities.graph.networkx_bridge import mappings_to_graph
 from oaklib.utilities.label_utilities import add_labels_to_object
 
+try:
```

Comment:
super minor but IMO it's cleaner not to wrap the defaultdict import as it's nonproblematic

it's just Iterator
https://stackoverflow.com/questions/72032032/importerror-cannot-import-name-iterable-from-collections-in-python

Reply:
in fact do do you need to wrap at all?

Doesn't this work in 3.9?
from collections.abc import Iterator

Reply:
You're right. It seems to work. Changing code